Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.11
```

Comment:
Consider pinning the Python version to a specific patch version (e.g., `python:3.11.1`) to ensure consistent builds and avoid unexpected changes when the base image is updated.